addons/website/models/ir_ui_view.py

```diff
@@ -13,12 +13,10 @@
 
 
 class view(osv.osv):
-    _inherit = "ir.ui.view"
+    _name = "ir.ui.view"
+    _inherit = ["ir.ui.view", "website.seo.metadata"]
```

Comment:
This change should be in a separate commit. Explain why you did this (not what you did, but why you did it).